text/0000-precise-pre-release-dependencies.md

```diff
@@ -0,0 +1,127 @@
+- Feature Name: precise_pre_release_cargo_update
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes extending `cargo update` to allow updates to pre-release versions when requested with `--precise`.
+For example, a `cargo` user would be able to call `cargo update -p dep --precise 0.1.1-pre0` as long as the version of `dep` requested by their project and its dependencies are semver compatible with `0.1.1`.
+This effectively splits the notion of compatibility in `cargo`.
+A pre-release version may considered compatible when `--precise`ly stated but will not be updated to automatically via a basic `cargo update`.
```

Comment:
```suggestion
A pre-release version may be considered compatible when `--precise`ly stated but it will not be automatically updated via a basic `cargo update`.
```

Reply:
Thanks for pointing this line out. I updated it in a slightly different way https://github.com/eopb/rfcs/commit/f5a1e9ff49979db29321362a1986b08c6e467345